src/components/UserAuth/UserAuth.module.css

```diff
@@ -0,0 +1,26 @@
+.userAuth {
+   display: flex;
+   background-color: #4dabf7;
+   border-radius: 0.7em;
+   height: 4em;
+   box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+   margin-right: 1em;
+}
+
+.authButton {
+   font-size: 1rem;
+   background: transparent;
+   border: none;
+   cursor: pointer;
+   display: flex;
+   align-items: center;
+   transition: 0.2s;
+   font-family: Roboto;
+   font-weight: 300;
+   white-space: nowrap;
+   margin: 0 1em;
+}
+
+.authButton:hover {
+   color: #a5d8ff;
+}
```

Comment:
Is it better to change the color of the text with buttons, or the background color? 
Try and look around the internet for what is more common, maybe even ask one of the UX :)

Reply:
the buttons are rounded, so it look a bit weird. I do can change the entire concept and make it like that.

Reply:
Change it :)